django/utils/http.py

```diff
@@ -362,10 +362,15 @@ def content_disposition_header(as_attachment, filename):
         disposition = "attachment" if as_attachment else "inline"
         try:
             filename.encode("ascii")
+            is_ascii = True
+        except UnicodeEncodeError:
+            is_ascii = False
+        # https://datatracker.ietf.org/doc/html/rfc9110#name-quoted-strings
+        if is_ascii and re.match(r"^[\t \x21-\x7e]*$", filename):
```

Comment:
Can you help me understand the regex and that it's equivalent to the quoted string?

Reply:
I've added more explanation to the commit message.  Does that help clarify?